mkdocs.yml

```diff
@@ -44,7 +44,7 @@ nav:
 
 
 markdown_extensions:
-    - toc:
-        permalink: 
+  - toc:
+      permalink: True
```

Comment:
This was set to a particular character, ¶ probably (or similar indicator of a permalink). How does the permalink look now?

Reply:
![obraz](https://user-images.githubusercontent.com/72821/217825694-1e12754a-7df8-42bb-b78a-62f17222a7a0.png)
Current permalink.

Looks good.

Reply:
It was meant to be this icon https://fontawesome.com/icons/link?s=solid&f=classic, I believe, but the Material theme doesn't support that icon in the same way as the old theme presumably did (I had no luck trying to force it). `True` just sets it to the theme default.